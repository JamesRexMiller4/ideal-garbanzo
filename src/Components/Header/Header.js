import React from 'react';
import play from '../../icons/play.svg';

const Header = () => {
  return ( 
    <header>
      <div className='logo-div'>
        <h1>Spectrum</h1>
        <img className='spectrum-icon' src={play} alt='play button'/>
      </div>
      <h2 className='app-title'>Ideal Garbanzo Service</h2>
    </header>
  );
}

export default Header;