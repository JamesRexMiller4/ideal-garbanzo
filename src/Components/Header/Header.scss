header {
  padding: 2rem;
}

.spectrum-icon {
  width: 1rem;
}

.logo-div {
  display: flex;
}

.app-title {
  font-style: italic;
}