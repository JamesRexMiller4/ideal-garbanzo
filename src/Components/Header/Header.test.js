import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';

describe('Header', () => {
  it('should render the snapshot', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});