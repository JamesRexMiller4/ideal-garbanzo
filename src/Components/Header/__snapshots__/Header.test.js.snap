// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render the snapshot 1`] = `
<header>
  <div
    className="logo-div"
  >
    <h1>
      Spectrum
    </h1>
    <img
      alt="play button"
      className="spectrum-icon"
      src="play.svg"
    />
  </div>
  <h2
    className="app-title"
  >
    Ideal Garbanzo Service
  </h2>
</header>
`;
